dttools/src/jx2env.c

```diff
@@ -0,0 +1,159 @@
+/*
+  Copyright (C) 2018- The University of Notre Dame This software is
+  distributed under the GNU General Public License.  See the file
+  COPYING for details.
+
+
+  Given a JSON object in a file, it prints...
+
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <errno.h>
+
+#include "getopt.h"
+#include "int_sizes.h"
+#include "jx.h"
+#include "jx_parse.h"
+#include "jx_print.h"
+#include "stringtools.h"
+#include "xxmalloc.h"
+
+void show_help(const char *exe) {
+	fprintf(stderr, "Usage:\n%s [--csh] input-file NAME=json.key.path [NAME=json.key.path ...]\n", exe);
```

Comment:
When I read this, it looks like you're binding a name by path rather than doing the underscore escape

Reply:
I am thinking of doing that to avoid name clashes. Let me submit the next commit.